tools/kbs-client/README.md

```diff
@@ -32,13 +32,13 @@ require fewer dependencies and so usually handy for CI) then you can pass the
 `sample_only` feature as:
 
 ```shell
-make -C ../../ cli CLI_FEATURES=sample_only
+make -C ../../kbs cli CLI_FEATURES=sample_only
```

Comment:
the change itself is OK but the comment above about `sample_only` looks outdated so we could update that too while we're at it. 

Default features come with `sample_only`. This section here should probably give instructions how to build a TEE specific flavor with: `make -C ../../kbs cli ATTESTER=<selected attester, e.g., tdx-attester`.

Reply:
The command in the diff is correct. Describing how to build with different attesters seems beyond the scope of this PR even though it's a pretty simple change.

Reply:
Indeed. I did not check the config before adding my comment. My recollection was that it was sample-only by default.